nextflow.config

```
@@ -56,6 +56,10 @@ params {
     filter_ssu        = null
     min_len_asv       = null
     max_len_asv       = null
+    filter_codons     = null
+    orf_start         = null
+    orf_end           = null
+    stop_codons       = null
```

Adding the default here is more helpful, I think, see also comment for the schema below that would require here 
```suggestion
    stop_codons       = "TAA,TAG"
```